mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -6395,3 +6396,27 @@ func containerClusterEnableK8sBetaApisCustomizeDiffFunc(d tpgresource.TerraformR
 
 	return nil
 }
+
+func containerClusterNodeVersionCustomizeDiff(_ context.Context,diff *schema.ResourceDiff, meta interface{}) error {
```

Comment:
Please add thorough unit tests for this customize diff func - similar to https://github.com/hashicorp/terraform-provider-google-beta/blob/dd55378df33eeea24904814063280c6057bb3e45/google-beta/services/compute/resource_compute_instance_template_internal_test.go#L156

Reply:
Added more tests